docker/wordpress/wp-content/plugins/a/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jetpack/a",
+  "description": "a",
+  "type": "wordpress-plugin",
+  "require": {
+    "automattic/jetpack-logo": "dev-try/custom-autoloader",
+    "automattic/autoloader": "1.0.0"
+  },
+  "repositories": [
+    {
+      "type": "path",
+      "url": "../jetpack/packages/logo",
+      "options": {
+        "symlink": true
+      }
+    },
+    {
+      "type": "path",
+      "url": "../jetpack/packages/autoloader",
+      "options": {
+        "symlink": false
+      }
+    }
+  ],
+  "scripts": {
```

Comment:
Are we going to have to specify these in each project/plugin that consumes one or more of our packages? If yes, I was hoping we could avoid that somehow.

Reply:
Good call. I am not sure how it could be avoided though. Since we are running after the install process completes. https://getcomposer.org/doc/articles/plugins.md looks promising but I haven't explored it

Reply:
I got it to work. Now we don't need this step any more.